.github/pull_request_template.md

```diff
@@ -6,6 +6,6 @@
 
 <!-- please check all items and add your own -->
 
-- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
+- [x] Read the contribution [guide](https://github.com/game-ci/cli/blob/main/CONTRIBUTING.md) and accept the [code](https://github.com/game-ci/cli/blob/main/CODE_OF_CONDUCT.md) of conduct
```

Comment:
Oh, cool. Didn't that was a thing! Learn something new everyday! :)

Reply:
Okay, I have tried it in the [documentation](https://github.com/game-ci/documentation) repo but it leads me to the following page https://github.com/game-ci/documentation/compare/blob/main/CONTRIBUTING.md.

screenshot:

![Image 1](https://user-images.githubusercontent.com/8685505/204977963-5563cace-99ba-4ade-8518-d06bb1a18dfd.png)

It works if you add one `../` to up one directory, so it became `../../blob/main/CONTRIBUTING.md`. Maybe it doesn't work for the fork but the branch? 😕

Reply:
Ah good point. Learned something new today as well.

Perhaps the hardcoded url is better in that case. This should probably be changed in all repos of GameCI (because I did it the same way everywhere).

Reply:
Yea let's go for that, since it'll always work

Reply:
Nice! I am merging this now! :)